openpype/modules/deadline/plugins/publish/submit_aftereffects_deadline.py

```diff
@@ -102,24 +103,18 @@ def get_job_info(self):
 
     def get_plugin_info(self):
         deadline_plugin_info = DeadlinePluginInfo()
-        context = self._instance.context
-        script_path = context.data["currentFile"]
 
         render_path = self._instance.data["expectedFiles"][0]
 
-        if len(self._instance.data["expectedFiles"]) > 1:
```

Comment:
Why stop relying on expected files. That seems wrong?

Reply:
Because it was replacing numbers with "#" only in cases where there were more expected files, not when there was only single one (but still with '0000' in its name). I think better option is to rely on presence of frame substring in a name.